x/operator/keeper/operator_slash_state.go

```diff
@@ -30,8 +30,12 @@ func (k *Keeper) UpdateOperatorSlashInfo(ctx sdk.Context, operatorAddr, avsAddr,
 		return errorsmod.Wrap(operatortypes.ErrSlashInfoExist, fmt.Sprintf("slashInfoKey:%s", slashInfoKey))
 	}
 	// check the validation of slash info
-	if slashInfo.SlashContract == "" {
-		return errorsmod.Wrap(operatortypes.ErrSlashInfo, fmt.Sprintf("err slashContract:%s", slashInfo.SlashContract))
+	getSlashContract, err := k.avsKeeper.GetAVSSlashContract(ctx, avsAddr)
```

Comment:
nit: Please rename this to `slashContract`

Reply:
thanks